.github/pull_request_template.md

```diff
@@ -1,3 +1,3 @@
-Before submitting a pull request, please read [CONTRIBUTING](./CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.
+Before submitting a pull request, please read [CONTRIBUTING](../CONTRIBUTING.md); pull requests that do not meet the criteria described there will not be merged. Note that this repository's CONTRIBUTING contains information specific to this repository, and is not the same as CONTRIBUTING for the void-packages repository.
```

Comment:
Are you sure you pushed the correct commit?

Reply:
I did it wrong.